src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/index.html.twig

```diff
@@ -26,6 +26,7 @@
 {% extends '@PrestaShop/Admin/layout.html.twig' %}
 
 {% block stylesheets %}
+  {{ parent() }}
```

Comment:
for now I only added it in orders page, but there are more. I can adjust the PR and add it everywhere I see (after the idea is accepted and if im not missing something here)